server/data/get-involved/council.json

```diff
@@ -0,0 +1,88 @@
r; color: rgb(198, 0, 120)\">(Bylaw 7.3)</a><span>&nbsp;</span>but First Year Representatives.</p><ol style=\" animation: 0.3s ease-in-out 0s 1 normal none running open; color: rgb(74, 78, 87); font-family: &quot;Open Sans&quot;; font-size: 16px\"><li><strong style=\" font-weight: 700\">Five (5) First Year Representatives,</strong><span>&nbsp;</span>which are open to all math students registered as first-year with the University, except for those in the Software Engineering program. First Year Representatives are elected once the term begins but before October 15 and shall serve until they are no longer a student in their first year, or until the end of the next Spring term,whichever is first.</li><li><strong style=\" font-weight: 700\">Three (3) Actuarial Science Representatives,</strong><span>&nbsp;</span>which are open to all math students in Actuarial Science or Mathematical Finance programs.</li><li><strong style=\" font-weight: 700\">Two (2) Statistics Representatives,</strong><span>&nbsp;</span>which are open to all math students in Statistics programs.</li><li><strong style=\" font-weight: 700\">Two (2) Pure Mathematics, Applied Mathematics, and Combinatorics and Optimization Representatives,</strong><span>&nbsp;</span>which are open to all math students in Pure Mathematics, Applied Mathematics, Combinatorics and Optimization, Mathematical Finance, or Computational Mathematics programs;</li><li><strong style=\" font-weight: 700\">Six (6) Computer Science Representatives,</strong><span>&nbsp;</span>which are open to all math students in programs offered by the David R. Cheriton School of Computer Science, as well as all students in Computational Mathematics programs;</li><li><strong style=\" font-weight: 700\">Four (4) Business Representatives,</strong><span>&nbsp;</span>which are open to all math students in Mathematics/Business programs, as well as all students in the Business Administration and Computer Science Double Degree program.</li><li><strong style=\" font-weight: 700\">One (1) Computing and Financial Management Representative,<span>&nbsp;</span></strong>which is open to all math students in the Computing and Financial Management program.</li><li><strong style=\" font-weight: 700\">Two (2) Software Engineering Representatives,</strong><span>&nbsp;</span>which are open to all math students in the Software Engineering program. The process to become a Software Engineering Representative is different from the other positions and is described in Bylaw 7.3.1.</li><li><strong style=\" font-weight: 700\">One (1) Teaching Representative,</strong><span>&nbsp;</span>which is open to all math students in the Mathematics/Teaching or Pure Mathematics/Teaching programs.</li><li><strong style=\" font-weight: 700\">Three (3) Mathematical Studies and Other Representatives,<span>&nbsp;</span></strong>which are open to all math students in Mathematical Studies programs and all math students not counted in one of the other constituencies.</li></ol>",
```

Comment:
Not really a requested change on this ticket, but we're going to need to find a way to handle editing these larger blocks of text.  I'm not sure what exactly the solution to that is though

Reply:
i envision using a rich text editor and somehow attaching it to the json editor for all editable values